main.cpp

```diff
@@ -37,27 +37,23 @@ int main(int argc, char **argv) {
 
         if (option.flag.short_name == "-u")
         {
-            url_file_provided = true;
-            load_urls_from_file(urls, option.value);
+            filename = option.value;
         }
 
         regex_mode = option.flag.short_name == "-r";
     }
 
-    if (!url_file_provided)
-    {
-        while (getline(std::cin, url))
-        {
-            urls.push_back(url);
-        }
+    if (filename.length() > 0) {
```

Comment:
Using c style brackets everywhere else so would prefer to be consistent

```
if (filename.length() > 0)
{
    ....
}
```

Reply:
How about I add a .clang-format to the repo, so we never have to worry about style inconsistencies again?